TEXT.md

```diff
@@ -164,9 +164,9 @@ let message = realstring.push_str(str1);
 - add `&str`s using `format!`
 
 ```rust
-let str1 = "hello ";
-let str2 = "world!";
-let message = format!("{}{}", str1, str2);
+let mut realstring = String::from("hello ");
```

Comment:
i think you meant for these changes to be in the above example (based on your other PRs!)